File: python_programming_basics/conditional_statements/e_05_godzilla_vs_kong.py
movie_budget = float(input())
people_count = int(input())
price_per_cloth = float(input())

decor = movie_budget * 0.1
price_for_clothing = people_count * price_per_cloth


if people_count > 150:
    price_for_clothing -= price_for_clothing * 0.1

total_sum = price_for_clothing + decor

if total_sum > movie_budget:
    print("Not enough money!")
    print(f"Wingard needs {total_sum - movie_budget:.2f} leva more.")

else:
    print("Action!")
    print(f"Wingard starts filming with {movie_budget - total_sum:.2f} leva left.")

File: JavaScript Basics/02. Conditional Statements/e-05-Godzilla-vs-Kong.js
function solve(budget, statics, clothingPrice) {
    let decorPrice = budget * 0.1;
    let totalClothing = statics * clothingPrice;

    if (statics > 150) {
        totalClothing *= 0.9;
    }

    let totalPrice = decorPrice + totalClothing;

    if (budget >= totalPrice) {
        let moneyLeft = budget - totalPrice;
        console.log("Action!");
        console.log(`Wingard starts filming with ${moneyLeft.toFixed(2)} leva left.`);
    
    } else {
        let moneyNeeded = totalPrice - budget;
        console.log("Not enough money!");
        console.log(`Wingard needs ${moneyNeeded.toFixed(2)} leva more.`);
    }
}

solve(20000, 120, 55.5);
solve(15437.62, 186, 57.99);
solve(9587.88, 222, 55.68);